Switch/Switch.module.less

```diff
@@ -49,6 +58,10 @@
 				:global(.spottable):focus & {
 					color: @agate-switch-selected-color;
 				}
+
+				.enact-locale-rtl({
+					right: @agate-switch-selected-icon-position-right;
```

Comment:
We don't need to add a variable for this. We could just use `@agate-switch-selected-icon-position-left`;
And you will need to add `position: absolute;` for `.icon` class since you are positioning from the right.

Actually, the result of the above code will revert the direction of the icon from
![image](https://user-images.githubusercontent.com/5037429/98204528-8d79b680-1f79-11eb-8e30-3c04ba3309ca.png)
to
![image](https://user-images.githubusercontent.com/5037429/98204546-966a8800-1f79-11eb-923c-77fc1dcd6761.png).

According to https://developer.mozilla.org/en-US/docs/Archive/B2G_OS/Firefox_OS_apps/Firefox_OS_in_Arabic, the switch button should be mirrored when RTL so we'd like to change the direction.

Reply:
thanks for the Arabic guide 👍